MapboxAndroidDemo/src/main/res/values/strings.xml

```diff
@@ -395,4 +396,10 @@
     <string name="retrieval_error_dialog_message">Looks like something went wrong with the log in process! Please try clearing your browser\'s cache/cookies and then sign back in.</string>
     <string name="retrieval_error_dialog_ok_button">Ok</string>
 
+    <!-- Share app intent -->
+    <string name="share_app_subject">Mapbox demo app for Android</string>
+    <string name="share_app_text">Hey, check out the great examples of what can be built with Mapbox:\n
```

Comment:
Passive voice here, use "Hey, check out these great examples showing what can be built using Mapbox." instead.